nntrainer/include/tensor.h

```diff
@@ -38,6 +38,30 @@
 namespace nntrainer {
 
 class LazyTensor;
+
+/**
+ * @struct External Loop Info for broadcasted info
+ * @brief External Loop Info for broadcasted iteration. Please refer to
+ * DISABLED_private_external_loop_n in unittest_nntrainer_tensor.
+ * @note This should better be implemented in iterator fashion before used
+ * extensively.
+ */
+struct ExternalLoopInfo {
```

Comment:
how about `broadcastInfo`?

Reply:
and `computeExternalLoop` to `computeBroadcastInfo`

Reply:
done